test/view/drop_while_view.cpp

```diff
@@ -0,0 +1,72 @@
+// cmcstl2 - A concept-enabled C++ standard library
+//
+//  Copyright Casey Carter
+//  Copyright Eric Niebler
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/caseycarter/cmcstl2
+//
+#include <stl2/view/drop_while.hpp>
+#include <stl2/view/iota.hpp>
+#include <stl2/view/take.hpp>
+#include <list>
+#include "../simple_test.hpp"
+
+namespace ranges = __stl2;
+
+namespace view {
+	using namespace ranges::view;
+	using view::ext::drop_while;
+}
+
+int main()
+{
+	{
+		auto rng0 = view::iota(10) | view::drop_while([](int i) { return i < 25; });
+		//  static_assert(range_cardinality<decltype(rng0)>::value == unknown);
+		static_assert(ranges::RandomAccessRange<decltype(rng0)>);
+		static_assert(!ranges::CommonRange<decltype(rng0)>);
+		static_assert(ranges::RandomAccessIterator<decltype(rng0.begin())>);
```

Comment:
This is still redundant with line 31. (We're not testing the range concepts here, and `RandomAccessRange`s always have `RandomAccessIterator`s.)